FestaListSorting/FestaListSortingv2.py

```diff
@@ -0,0 +1,118 @@
+import operator
+import datetime
+import os
+import re
+
+# 페스타 이벤트 정보를 담는 클래스
+class FeastaEvent:
+    def __init__(self, name, date, host):
+        self.name = name
+        self.date = date
+        self.host = host
+
+    # 생성할 파일 이름
+    # 요구사항대로 특수문자를 빼고 띄어쓰기를 _ 로 대치한다.
+    # 정규식을 이용하여 대치했다.
+    def getFileName(self):
+        regExp = '[-=+,#/\?:^$.@*\"※~&%ㆍ!』\\‘|\(\)\[\]\<\>`\'…》]'
+        file_name = re.sub(regExp, '', event.name).strip().replace("  ", " ").replace(" ", "_")
+        return file_name
+
+    # 생성할 파일 경로 (YYYY/mmdd/파일이름.md) 를 반환
+    def getFileNameWithPath(self):
+        file_path = self.date[:4]
+        month_date = self.date[6:12].replace(".", "").replace(" ", "").strip() 
+        file_name_path = self.date[:4] + "/" + month_date + "/" + month_date + "_" + self.getFileName() + ".md"
+        return file_name_path
+        
+    # 어째서인지 이모지를 콘솔에 찍으면 깨지는 버그가 있어서
+    # 그냥 아예 temp.txt에 생성하도록 변경하였다.
+    def printEvent(self):
+        # open 을 해서 없는 파일이면 새로 생성한다.
+        # a는 내용 추가, r은 읽기전용, w은 쓰기이다.
+        # w대신 a를 쓴 이유는 내용을 계속 추가해야하는데 w로 쓰면 덮어씌워져서 이전 반복문에서 추가한 내용이 사라지기 때문.
+        f= open("temp.txt", "a")
+        content = "### " + self.name + "\n\n"
+        content += "- **⏰ 일시** : " + self.date + "\n"
+        content += "- **💁 주최** : " + self.host + "\n"
+        content += "- **📝 총평** : 추가 예정\n"
+        content += "- **👀 리뷰** : [상세 리뷰 보러 가기](" + self.getFileNameWithPath() + ")\n\n\n"
```

Comment:
\n 대신에 os.linesep 사용 추천
가독성은 지금 형식이 제일 좋지만 성능이 중요시 될때는  ''.join() 사용 추천
PEP 8 : In performance sensitive parts of the library, the ''.join() form should be used instead. This will ensure that concatenation occurs in linear time across various implementations.